cognite/client/_api/files.py

```diff
@@ -545,6 +545,9 @@ def upload_bytes(
                 >>> c = CogniteClient()
                 >>> res = c.files.upload_bytes(b"some content", name="my_file", asset_ids=[1,2,3])
         """
+        if isinstance(content, str):
+            content = content.encode("utf-8")
```

Comment:
Is this Windows `requests` bug? Great catch anyway!

Reply:
actually, based on the bug it seems to also happen on unix systems.